webapp/packages/core-blocks/src/FormControls/Checkbox.tsx

```diff
@@ -87,21 +133,30 @@ export const Checkbox: CheckboxType = observer(function Checkbox({
 
   const checked = state ? state[name] : checkedControlled;
 
-  return styled(useStyles(baseFormControlStyles, styles))(
-    <field as="div" className={className} {...use({ long })}>
-      <field-label as="div">{children}</field-label>
+  return styled(useStyles(baseFormControlStyles, checkboxStyles, checkboxMod.primary, disabled &&
+     checkboxState.disabled, checked && checkboxState.checked))(
+    <field as="div" className={className}>
```

Comment:
You should create `FieldCheckbox` component because this change brokes form's checkboxes